toolkit/legend/src/main/java/com/arcgismaps/toolkit/legend/theme/Theme.kt

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2025 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.arcgismaps.toolkit.legend.theme
+
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Immutable
+import androidx.compose.ui.text.TextStyle
+
+/**
+ * Typography styling properties for the Legend composable.
+ *
+ * @param layerName The text style for the layer name.
+ * @param layerTitle The text style for the layer title.
+ *
+ * @since 200.7.0
+ */
+@Immutable
+public data class Typography internal constructor(
+    val layerName : TextStyle,
+    val layerTitle: TextStyle,
+)
+
+/**
+ * Default values for the Legend composable.
+ *
+ * @since 200.7.0
+ */
+public object LegendDefaults {
+
+    /**
+     * Creates default typography values for the Legend composable.
+     *
+     * @param layerName The text style for the layer name.
+     * @param layerTitle The text style for the layer title.
+     */
+    @Composable
+    public fun typography(
+        layerName: TextStyle = MaterialTheme.typography.titleLarge,
+        layerTitle: TextStyle = MaterialTheme.typography.titleMedium,
```

Comment:
This should be renamed to legendInfoName, also can you change the default value to `bodyMedium`
```suggestion
        legendInfoName: TextStyle = MaterialTheme.typography.bodyMedium,
```